sys/embunit/TestRunner.c

```diff
@@ -47,7 +47,7 @@ static void TestRunner_startTest(TestListner* self,Test* test)
 {
     (void)self;
     (void)test;
-    stdimpl_print(".");
+    stdimpl_print(".\n");
```

Comment:
comma should be followed by whitespace  3 line above

i dont care but the static tests do

Reply:
From OP:

> This PR is not intended to get merged.